src/templates/player.js

```diff
@@ -1,6 +1,7 @@
 export default (id, ariaLabel = '') => {
     return (
         `<div id="${id}" class="jwplayer jw-reset jw-state-setup" tabindex="0" aria-label="${ariaLabel}" role="application">` +
+            `<div class="jw-aspect jw-reset"></div>` +
```

Comment:
On a non-outstream floating player, we will not show the poster image on the floating player if we do this. I think it is fine as it is though.

Reply:
Looks like you're right. Not sure why though?

Reply:
We use `jw-aspect` to display the poster, so if we have this outside of the `jw-wrapper`, then it won't show on the floating player.